main.go

```diff
@@ -1,14 +1,77 @@
 package main
 
 import (
-	"ava-go/bot"
+	"ava-go/handlers"
+	"errors"
 	"log"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/bwmarrin/discordgo"
 )
 
+func start() (err error) {
+	token, ok := os.LookupEnv("BOT_TOKEN")
+	if !ok {
+		return errors.New("couldn't find environment variable $BOT_TOKEN")
+	}
+
+	goBot, err := discordgo.New("Bot " + token)
+	if err != nil {
+		return errors.New("bot auth: " + err.Error())
+	}
+
+	// declare intents (needed to be able to get member info)
+	goBot.Identify.Intents = discordgo.MakeIntent(discordgo.IntentsAll)
+	goBot.AddHandler(handlers.MessageHandler)
+	goBot.AddHandler(handlers.VoiceStateHandler)
+
+	// add slash command handlers
+	goBot.AddHandler(func(s *discordgo.Session, i *discordgo.InteractionCreate) {
+		if h, ok := handlers.CommandHandlers[i.ApplicationCommandData().Name]; ok {
+			h(s, i)
+		}
+	})
+
+	if err = goBot.Open(); err != nil {
+		return errors.New("bot open: " + err.Error())
+	}
+
+	log.Println("bot is running!")
+
+	// add commands
+	registeredCommands := make([]*discordgo.ApplicationCommand, len(handlers.Commands))
+	for i, v := range handlers.Commands {
+		cmd, err := goBot.ApplicationCommandCreate(goBot.State.User.ID, "379276406326165515", v)
+		if err != nil {
+			return errors.New("cannot create command " + v.Name + ": " + err.Error())
+		}
+		registeredCommands[i] = cmd
+	}
+
+	defer goBot.Close()
+
+	stop := make(chan os.Signal, 1)
+	signal.Notify(stop, os.Interrupt)    // stop on ctrl-C
+	signal.Notify(stop, syscall.SIGTERM) // stop on docker stop
+	<-stop
+
+	// clean up added commands
+	for _, v := range registeredCommands {
```

Comment:
another single-character variable name that should probably be cleaned up (maybe `command`?)